scripts/legislation/africa/namibia_english.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+
+function remove_all_text_before_first_header {
+  sed -n '/^58. Short title and commencement/,$p' | \
+    sed -n '/^PART I/,$p'
+}
+
+function prefix_article_numbers_with_article_literal {
+  sed -E 's/^([0-9]+\.)/Article \1/'
+}
+
+function replace_parentheses_around_article_delimiters_with_square_brackets {
+  sed -E 's/^\(([A-Za-z0-9]+)\)/[\1]/'
+}
+
+function amend_errors_in_headers {
+  sed -E 's/^(Section [0-9]+ - [A-Za-z,’ ]+)[^a-z]+Article ([0-9]+)(er)?/\1\n\n(\2)/' | \
+    sed -E 's/^(PART )I/\1I -/' | \
+    sed -E 's/^(PART I - DEFINITIONS.*) Definitions/\1\n\nDefinitions/' | \
+    sed -E 's/^(PART )I -I/\1II -/' | \
+    sed -E 's/^(PART II - PRINCIPLES.*) Principles/\1\n\nPrinciples/' | \
+    sed -E 's/^(PART )II -I/\1III -/' | \
+    sed -E 's/^(PART III - GENERAL.*) Functions/\1\n\nFunctions/' | \
+    sed -E 's/^(PART )I -V/\1IV -/' | \
+    sed -E 's/^(PART IV - SUSTAINABLE.*) Establishment/\1\n\nEstablishment/' | \
+    sed -E 's/^(PART )V/\1V -/' | \
+    sed -E 's/^(PART V - ENVIRONMENTAL.*) Appointment/\1\n\nAppointment/' | \
+    sed -E 's/^(PART )V -I/\1VI -/' | \
+    sed -E 's/^(PART VI - ENVIRONMENTAL.*) Objects/\1\n\nObjects/' | \
+    sed -E 's/^(PART )VI -I/\1VII -/' | \
+    sed -E 's/^(PART VII - ENVIRONMENTAL.*) Listing/\1\n\nListing/' | \
+    sed -E 's/^(PART )VII -I/\1VIII -/' | \
+    sed -E 's/^(PART VIII - ENVIRONMENTAL.*) Application/\1\n\nApplication/' | \
+    sed -E 's/^(PART )I -X/\1IX -/' | \
+    sed -E 's/^(PART IX - SPECIAL.*) Consultation/\1\n\nConsultation/' 
+}
+
+function remove_margin_headers {
+    sed -E 's/No. 3966 Government Gazette 27 December 2007//g' | \
+    sed -E 's/Act No. 7, 2007 ENVIRONMENTAL MANAGEMENT ACT, 2007//g' | \
+    sed -E 's/Government Gazette 27 December 2007 No. 3966 //g' 
+}
+
+function identify_leftover_articles {
```

Comment:
`remove_leftover_article_literals` would probably be a more appropriate function name.